app/models/child.rb

```diff
@@ -68,6 +68,14 @@ def attendances
     Attendance.joins(:child_approval).where(child_approvals: { child: self })
   end
 
+  def absences(filter_date)
```

Comment:
[QUESTION] This method looks like it returns a presentation `x of n` required on the UI. Is there an intention of being able to reuse it for non UI operations?